src/models/data_table.ts

```diff
@@ -33,7 +33,7 @@ export default class DataTable {
     return copy
   }
 
-  rowsHash(): Dictionary<any> {
+  rowsHash(): Record<string, any> {
```

Comment:
As `this.raw()` returns a `string[][]`, would it be possible for rowsHash to return a `Record<string, string>` instead of `Record<string, any>`?

Reply:
You're right, good catch - will change.